examples/rust/poseidon2/run.sh

```diff
@@ -0,0 +1 @@
+echo "to be included in future CI runs"
```

Comment:
Lets not forget about this

Reply:
Who should be responsible for that?

Reply:
@jeremyfelder: can u take care of it? I spoke to @yshekel, and he said that the CI needs to run to ensure that the examples are consistent with any future changes. It makes sense to me. But im not the expert at that :)

Reply:
Done in commit [a9a368a](https://github.com/ingonyama-zk/icicle/pull/681/commits/a9a368aca2badf5480194935fb17e1c33cdc8ad5)